### DISCUSSION TITLE
Training loss stuck and not decreasing at all

### DISCUSSION THREAD

**Discussion by moonlight-ace** [moonlight-ace/AnalysticFinancial#15] [Q&A]
My model trains but the loss stays constant.
Any suggestions?
